lib/Mail/Milter/Authentication/Handler/SPF.pm

```diff
@@ -372,5 +379,8 @@ Implements the SPF standard checks.
             "best_guess"               : 0,             | Fallback to Org domain for SPF checks
                                                         | if result is none.
             "spfu_detection"           : 0              | Add some mitigation for SPF upgrade attacks
+                                                        | 0 = off (default)
+                                                        | 1 = mitigate
+                                                        | 2 = report only (any value other than 1)
```

Comment:
What if a user puts `3` in their config, and sometime down the line you want another option.  Then someone is going to get a surprise.  At minimum i'd suggest removing the section in parenthesis.